README.md

```diff
@@ -123,7 +124,8 @@ Command     | Aliases         | Subcommands
 `method`    | `m`             | `instance`
 `next`      | `n`             |
 `pry`       |                 |
-`quit`      | `q`             |
+`quit`      | `q`             | `!`
```

Comment:
I guess you're adding `!` as a subcommand because `quit !` and `q !` (command followed by a space and then `!`) works?

Reply:
Yes, it works. Also, there is subcommand `unconditionally`(with the same behavior as `!`). I can add this too.
And the same subcommands for a command `continue`.

Reply:
It is an option for command, not a sub-command. Canceled this change.